code/datums/martial.dm

```diff
@@ -3,7 +3,7 @@
   * Martial Arts
   *
   * Martial arts change how human unarmed attacks effect other humans, and allow for strings of attacks on the same target to have special effects
-  * Martial arts also can block melee or ranged attacks using block_chance and deflect_chance variables
+  *   arts also can block melee or ranged attacks using block_chance and deflect_chance variables
```

Comment:
Why tho

Reply:
uh, yeah, totally intended to do that, and i'm not just a bull in a china shop with all this code